ui/lib/core/addon/components/info-table-row.js

```diff
@@ -75,4 +77,12 @@ export default class InfoTableRowComponent extends Component {
       }
     }
   }
+
+  @action
```

Comment:
Okay, don't hate me on this. But... this solution (calcuatling on did-insert with an action) was originally needed so this would be done for all use cases—depdending on the size of the screen width and label length the truncation would happen. However, becuase we're not calculating anything and are more-or-less passing in a param here, let's just do this as you would normally would: pass in something like `@truncateValue=true`, from the instance of `InfoTableRow` on the `paths.hbs` file and conditionally render the class that way. Let me know if that doesn't make sense.

Reply:
That makes sense to me and I like that better! Will update now!